arch/arm64/boot/dts/qcom/msm8998-yoshino-common.dtsi

```diff
@@ -205,7 +205,7 @@
 			watchdog_delay_ms = <3000>;
 			charger_supported = <0>;
```

Comment:
Sorry to bother you, but is this flag for Qnovo?

Reply:
Unfortunately I don't know; looking at the driver all this does is allowing to set a bit in the touchscreen. Whether it is to mitigate frequency issues during charging, display a low-power charge value/icon/animation on the screen, or make the display more aware of the battery and adapt power usage to that...

There also seems to be no documentation about clearpad in our kernel.